scripts/format.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const fs = require('fs')
+const summarize = require('tap-markdown')
+
+exports.mdFormatResults = (resultsFile) => {
```

Comment:
What does `md` stand for?

Reply:
Ah, its markdown. Maybe we can say `markdownFormatResults` instead, to reduce some cognitive load? :D